themes/src/main/resources/theme/keycloak-preview/account/resources/styles.css

```diff
@@ -301,11 +301,35 @@ p.description {
   position: absolute;
   right: 10px;
 }
+.list-view-pf-additional-info .fa-user{
+  margin-right: 5px;
+}
+.linked-account-list .list-view-pf .list-group-item-heading {
+    -ms-flex: 1 0 calc(40% - 20px);
```

Comment:
no need to add -ms-flex.
PatternFly uses Autoprefixer to deal with CSS vendor prefixes.
Do not add vender prefixes to your code, Autoprefixer will do it for you on build.